build.yaml

```diff
@@ -90,24 +90,52 @@ common_cli:
     #
```

Comment:
Looking at the comment above this, it seems this has long stopped being the '"first-line".